2021-03-06 00:18:05,657 [INFO] from play.api.db.DefaultDBApi in play-dev-mode-akka.actor.default-dispatcher-14 - Database [default] initialized
2021-03-06 00:18:05,658 [INFO] from play.api.db.HikariCPConnectionPool in play-dev-mode-akka.actor.default-dispatcher-14 - Creating Pool for datasource 'default'
2021-03-06 00:18:06,625 [INFO] from play.api.http.EnabledFilters in play-dev-mode-akka.actor.default-dispatcher-14 - Enabled Filters (see <https://www.playframework.com/documentation/latest/Filters>):

    play.filters.csrf.CSRFFilter
    play.filters.headers.SecurityHeadersFilter
    play.filters.hosts.AllowedHostsFilter

2021-03-06 00:18:06,668 [INFO] from play.api.Play in play-dev-mode-akka.actor.default-dispatcher-14 - Application started (Dev) (no global state)
2021-03-06 00:57:01,745 [INFO] from play.api.db.HikariCPConnectionPool in application-akka.actor.internal-dispatcher-9 - Shutting down connection pool.
2021-03-06 00:57:01,986 [INFO] from play.api.db.DefaultDBApi in play-dev-mode-akka.actor.default-dispatcher-24 - Database [default] initialized
2021-03-06 00:57:01,987 [INFO] from play.api.db.HikariCPConnectionPool in play-dev-mode-akka.actor.default-dispatcher-24 - Creating Pool for datasource 'default'
2021-03-06 00:57:02,193 [INFO] from play.api.http.EnabledFilters in play-dev-mode-akka.actor.default-dispatcher-24 - Enabled Filters (see <https://www.playframework.com/documentation/latest/Filters>):

    play.filters.csrf.CSRFFilter
    play.filters.headers.SecurityHeadersFilter
    play.filters.hosts.AllowedHostsFilter

2021-03-06 00:57:02,202 [INFO] from play.api.Play in play-dev-mode-akka.actor.default-dispatcher-24 - Application started (Dev) (no global state)
2021-03-06 01:07:52,637 [INFO] from play.api.db.HikariCPConnectionPool in application-akka.actor.internal-dispatcher-8 - Shutting down connection pool.
2021-03-06 01:07:52,861 [INFO] from play.api.db.DefaultDBApi in play-dev-mode-akka.actor.default-dispatcher-41 - Database [default] initialized
2021-03-06 01:07:52,861 [INFO] from play.api.db.HikariCPConnectionPool in play-dev-mode-akka.actor.default-dispatcher-41 - Creating Pool for datasource 'default'
2021-03-06 01:07:53,073 [INFO] from play.api.http.EnabledFilters in play-dev-mode-akka.actor.default-dispatcher-41 - Enabled Filters (see <https://www.playframework.com/documentation/latest/Filters>):

    play.filters.csrf.CSRFFilter
    play.filters.headers.SecurityHeadersFilter
    play.filters.hosts.AllowedHostsFilter

2021-03-06 01:07:53,079 [INFO] from play.api.Play in play-dev-mode-akka.actor.default-dispatcher-41 - Application started (Dev) (no global state)
2021-03-06 01:10:27,537 [INFO] from play.api.db.HikariCPConnectionPool in application-akka.actor.internal-dispatcher-7 - Shutting down connection pool.
2021-03-06 01:10:27,745 [INFO] from play.api.db.DefaultDBApi in play-dev-mode-akka.actor.default-dispatcher-48 - Database [default] initialized
2021-03-06 01:10:27,745 [INFO] from play.api.db.HikariCPConnectionPool in play-dev-mode-akka.actor.default-dispatcher-48 - Creating Pool for datasource 'default'
2021-03-06 01:10:27,951 [INFO] from play.api.http.EnabledFilters in play-dev-mode-akka.actor.default-dispatcher-48 - Enabled Filters (see <https://www.playframework.com/documentation/latest/Filters>):

    play.filters.csrf.CSRFFilter
    play.filters.headers.SecurityHeadersFilter
    play.filters.hosts.AllowedHostsFilter

2021-03-06 01:10:27,957 [INFO] from play.api.Play in play-dev-mode-akka.actor.default-dispatcher-48 - Application started (Dev) (no global state)
2021-03-06 01:15:26,773 [INFO] from play.api.db.HikariCPConnectionPool in application-akka.actor.internal-dispatcher-7 - Shutting down connection pool.
2021-03-06 01:15:27,003 [INFO] from play.api.db.DefaultDBApi in play-dev-mode-akka.actor.default-dispatcher-56 - Database [default] initialized
2021-03-06 01:15:27,003 [INFO] from play.api.db.HikariCPConnectionPool in play-dev-mode-akka.actor.default-dispatcher-56 - Creating Pool for datasource 'default'
2021-03-06 01:15:27,220 [INFO] from play.api.http.EnabledFilters in play-dev-mode-akka.actor.default-dispatcher-56 - Enabled Filters (see <https://www.playframework.com/documentation/latest/Filters>):

    play.filters.csrf.CSRFFilter
    play.filters.headers.SecurityHeadersFilter
    play.filters.hosts.AllowedHostsFilter

2021-03-06 01:15:27,228 [INFO] from play.api.Play in play-dev-mode-akka.actor.default-dispatcher-56 - Application started (Dev) (no global state)
2021-03-06 01:21:11,998 [INFO] from play.api.db.HikariCPConnectionPool in application-akka.actor.internal-dispatcher-7 - Shutting down connection pool.
2021-03-06 01:21:12,258 [INFO] from play.api.db.DefaultDBApi in play-dev-mode-akka.actor.default-dispatcher-64 - Database [default] initialized
2021-03-06 01:21:12,258 [INFO] from play.api.db.HikariCPConnectionPool in play-dev-mode-akka.actor.default-dispatcher-64 - Creating Pool for datasource 'default'
2021-03-06 01:21:12,503 [INFO] from play.api.http.EnabledFilters in play-dev-mode-akka.actor.default-dispatcher-64 - Enabled Filters (see <https://www.playframework.com/documentation/latest/Filters>):

    play.filters.csrf.CSRFFilter
    play.filters.headers.SecurityHeadersFilter
    play.filters.hosts.AllowedHostsFilter

2021-03-06 01:21:12,508 [INFO] from play.api.Play in play-dev-mode-akka.actor.default-dispatcher-64 - Application started (Dev) (no global state)
2021-03-06 01:22:33,464 [INFO] from play.api.db.HikariCPConnectionPool in application-akka.actor.internal-dispatcher-7 - Shutting down connection pool.
2021-03-06 01:22:33,726 [INFO] from play.api.db.DefaultDBApi in play-dev-mode-akka.actor.default-dispatcher-72 - Database [default] initialized
2021-03-06 01:22:33,726 [INFO] from play.api.db.HikariCPConnectionPool in play-dev-mode-akka.actor.default-dispatcher-72 - Creating Pool for datasource 'default'
2021-03-06 01:22:33,921 [INFO] from play.api.http.EnabledFilters in play-dev-mode-akka.actor.default-dispatcher-72 - Enabled Filters (see <https://www.playframework.com/documentation/latest/Filters>):

    play.filters.csrf.CSRFFilter
    play.filters.headers.SecurityHeadersFilter
    play.filters.hosts.AllowedHostsFilter

2021-03-06 01:22:33,925 [INFO] from play.api.Play in play-dev-mode-akka.actor.default-dispatcher-72 - Application started (Dev) (no global state)
2021-03-06 01:27:56,587 [ERROR] from play.api.http.DefaultHttpErrorHandler in play-dev-mode-akka.actor.default-dispatcher-82 - 

! @7j1o5fka8 - Internal server error, for (GET) [/recetas/search/usuario/id/1] ->
 
play.sbt.PlayExceptions$CompilationException: Compilation error[type mismatch;
 found   : java.util.List[models.Ingrediente]
 required: Int]
	at play.sbt.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:34)
	at play.sbt.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:34)
	at scala.Option.map(Option.scala:230)
	at play.sbt.run.PlayReload$.$anonfun$taskFailureHandler$1(PlayReload.scala:40)
	at scala.Option.map(Option.scala:230)
	at play.sbt.run.PlayReload$.taskFailureHandler(PlayReload.scala:35)
	at play.sbt.run.PlayReload$.compileFailure(PlayReload.scala:28)
	at play.sbt.run.PlayReload$.$anonfun$compile$3(PlayReload.scala:63)
	at scala.util.Either$LeftProjection.map(Either.scala:573)
	at play.sbt.run.PlayReload$.compile(PlayReload.scala:63)
	at play.sbt.run.PlayRun$.$anonfun$playRunTask$4(PlayRun.scala:78)
	at play.runsupport.Reloader.$anonfun$reload$1(Reloader.scala:496)
	at play.runsupport.Reloader$$anon$1.run(Reloader.scala:57)
	at java.security.AccessController.doPrivileged(Native Method)
	at play.runsupport.Reloader$.play$runsupport$Reloader$$withReloaderContextClassLoader(Reloader.scala:63)
	at play.runsupport.Reloader.reload(Reloader.scala:496)
	at play.core.server.DevServerStart$$anon$1.get(DevServerStart.scala:141)
	at play.core.server.AkkaHttpServer.handleRequest(AkkaHttpServer.scala:302)
	at play.core.server.AkkaHttpServer.$anonfun$createServerBinding$1(AkkaHttpServer.scala:224)
	at akka.stream.impl.fusing.MapAsync$$anon$30.onPush(Ops.scala:1284)
	at akka.stream.impl.fusing.GraphInterpreter.processPush(GraphInterpreter.scala:541)
	at akka.stream.impl.fusing.GraphInterpreter.execute(GraphInterpreter.scala:423)
	at akka.stream.impl.fusing.GraphInterpreterShell.runBatch(ActorGraphInterpreter.scala:625)
	at akka.stream.impl.fusing.GraphInterpreterShell$AsyncInput.execute(ActorGraphInterpreter.scala:502)
	at akka.stream.impl.fusing.GraphInterpreterShell.processEvent(ActorGraphInterpreter.scala:600)
	at akka.stream.impl.fusing.ActorGraphInterpreter.akka$stream$impl$fusing$ActorGraphInterpreter$$processEvent(ActorGraphInterpreter.scala:769)
	at akka.stream.impl.fusing.ActorGraphInterpreter$$anonfun$receive$1.applyOrElse(ActorGraphInterpreter.scala:784)
	at akka.actor.Actor.aroundReceive(Actor.scala:537)
	at akka.actor.Actor.aroundReceive$(Actor.scala:535)
	at akka.stream.impl.fusing.ActorGraphInterpreter.aroundReceive(ActorGraphInterpreter.scala:691)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:577)
	at akka.actor.ActorCell.invoke(ActorCell.scala:547)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:270)
	at akka.dispatch.Mailbox.run(Mailbox.scala:231)
	at akka.dispatch.Mailbox.exec(Mailbox.scala:243)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
2021-03-06 01:29:19,358 [ERROR] from play.api.http.DefaultHttpErrorHandler in play-dev-mode-akka.actor.default-dispatcher-83 - 

! @7j1o5kcpk - Internal server error, for (GET) [/recetas/search/usuario/id/1] ->
 
play.sbt.PlayExceptions$CompilationException: Compilation error[type mismatch;
 found   : java.util.List[models.Ingrediente]
 required: Int]
	at play.sbt.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:34)
	at play.sbt.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:34)
	at scala.Option.map(Option.scala:230)
	at play.sbt.run.PlayReload$.$anonfun$taskFailureHandler$1(PlayReload.scala:40)
	at scala.Option.map(Option.scala:230)
	at play.sbt.run.PlayReload$.taskFailureHandler(PlayReload.scala:35)
	at play.sbt.run.PlayReload$.compileFailure(PlayReload.scala:28)
	at play.sbt.run.PlayReload$.$anonfun$compile$3(PlayReload.scala:63)
	at scala.util.Either$LeftProjection.map(Either.scala:573)
	at play.sbt.run.PlayReload$.compile(PlayReload.scala:63)
	at play.sbt.run.PlayRun$.$anonfun$playRunTask$4(PlayRun.scala:78)
	at play.runsupport.Reloader.$anonfun$reload$1(Reloader.scala:496)
	at play.runsupport.Reloader$$anon$1.run(Reloader.scala:57)
	at java.security.AccessController.doPrivileged(Native Method)
	at play.runsupport.Reloader$.play$runsupport$Reloader$$withReloaderContextClassLoader(Reloader.scala:63)
	at play.runsupport.Reloader.reload(Reloader.scala:496)
	at play.core.server.DevServerStart$$anon$1.get(DevServerStart.scala:141)
	at play.core.server.AkkaHttpServer.handleRequest(AkkaHttpServer.scala:302)
	at play.core.server.AkkaHttpServer.$anonfun$createServerBinding$1(AkkaHttpServer.scala:224)
	at akka.stream.impl.fusing.MapAsync$$anon$30.onPush(Ops.scala:1284)
	at akka.stream.impl.fusing.GraphInterpreter.processPush(GraphInterpreter.scala:541)
	at akka.stream.impl.fusing.GraphInterpreter.execute(GraphInterpreter.scala:423)
	at akka.stream.impl.fusing.GraphInterpreterShell.runBatch(ActorGraphInterpreter.scala:625)
	at akka.stream.impl.fusing.GraphInterpreterShell$AsyncInput.execute(ActorGraphInterpreter.scala:502)
	at akka.stream.impl.fusing.GraphInterpreterShell.processEvent(ActorGraphInterpreter.scala:600)
	at akka.stream.impl.fusing.ActorGraphInterpreter.akka$stream$impl$fusing$ActorGraphInterpreter$$processEvent(ActorGraphInterpreter.scala:769)
	at akka.stream.impl.fusing.ActorGraphInterpreter$$anonfun$receive$1.applyOrElse(ActorGraphInterpreter.scala:784)
	at akka.actor.Actor.aroundReceive(Actor.scala:537)
	at akka.actor.Actor.aroundReceive$(Actor.scala:535)
	at akka.stream.impl.fusing.ActorGraphInterpreter.aroundReceive(ActorGraphInterpreter.scala:691)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:577)
	at akka.actor.ActorCell.invoke(ActorCell.scala:547)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:270)
	at akka.dispatch.Mailbox.run(Mailbox.scala:231)
	at akka.dispatch.Mailbox.exec(Mailbox.scala:243)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
2021-03-06 01:30:05,044 [ERROR] from play.api.http.DefaultHttpErrorHandler in play-dev-mode-akka.actor.default-dispatcher-83 - 

! @7j1o5n2eo - Internal server error, for (GET) [/recetas/search/usuario/id/1] ->
 
play.sbt.PlayExceptions$CompilationException: Compilation error[type mismatch;
 found   : java.util.List[models.Ingrediente]
 required: Int]
	at play.sbt.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:34)
	at play.sbt.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:34)
	at scala.Option.map(Option.scala:230)
	at play.sbt.run.PlayReload$.$anonfun$taskFailureHandler$1(PlayReload.scala:40)
	at scala.Option.map(Option.scala:230)
	at play.sbt.run.PlayReload$.taskFailureHandler(PlayReload.scala:35)
	at play.sbt.run.PlayReload$.compileFailure(PlayReload.scala:28)
	at play.sbt.run.PlayReload$.$anonfun$compile$3(PlayReload.scala:63)
	at scala.util.Either$LeftProjection.map(Either.scala:573)
	at play.sbt.run.PlayReload$.compile(PlayReload.scala:63)
	at play.sbt.run.PlayRun$.$anonfun$playRunTask$4(PlayRun.scala:78)
	at play.runsupport.Reloader.$anonfun$reload$1(Reloader.scala:496)
	at play.runsupport.Reloader$$anon$1.run(Reloader.scala:57)
	at java.security.AccessController.doPrivileged(Native Method)
	at play.runsupport.Reloader$.play$runsupport$Reloader$$withReloaderContextClassLoader(Reloader.scala:63)
	at play.runsupport.Reloader.reload(Reloader.scala:496)
	at play.core.server.DevServerStart$$anon$1.get(DevServerStart.scala:141)
	at play.core.server.AkkaHttpServer.handleRequest(AkkaHttpServer.scala:302)
	at play.core.server.AkkaHttpServer.$anonfun$createServerBinding$1(AkkaHttpServer.scala:224)
	at akka.stream.impl.fusing.MapAsync$$anon$30.onPush(Ops.scala:1284)
	at akka.stream.impl.fusing.GraphInterpreter.processPush(GraphInterpreter.scala:541)
	at akka.stream.impl.fusing.GraphInterpreter.execute(GraphInterpreter.scala:423)
	at akka.stream.impl.fusing.GraphInterpreterShell.runBatch(ActorGraphInterpreter.scala:625)
	at akka.stream.impl.fusing.GraphInterpreterShell$AsyncInput.execute(ActorGraphInterpreter.scala:502)
	at akka.stream.impl.fusing.GraphInterpreterShell.processEvent(ActorGraphInterpreter.scala:600)
	at akka.stream.impl.fusing.ActorGraphInterpreter.akka$stream$impl$fusing$ActorGraphInterpreter$$processEvent(ActorGraphInterpreter.scala:769)
	at akka.stream.impl.fusing.ActorGraphInterpreter$$anonfun$receive$1.applyOrElse(ActorGraphInterpreter.scala:784)
	at akka.actor.Actor.aroundReceive(Actor.scala:537)
	at akka.actor.Actor.aroundReceive$(Actor.scala:535)
	at akka.stream.impl.fusing.ActorGraphInterpreter.aroundReceive(ActorGraphInterpreter.scala:691)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:577)
	at akka.actor.ActorCell.invoke(ActorCell.scala:547)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:270)
	at akka.dispatch.Mailbox.run(Mailbox.scala:231)
	at akka.dispatch.Mailbox.exec(Mailbox.scala:243)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
2021-03-06 01:30:07,892 [ERROR] from play.api.http.DefaultHttpErrorHandler in play-dev-mode-akka.actor.default-dispatcher-81 - 

! @7j1o5n6kd - Internal server error, for (GET) [/recetas/search/usuario/id/1] ->
 
play.sbt.PlayExceptions$CompilationException: Compilation error[type mismatch;
 found   : java.util.List[models.Ingrediente]
 required: Int]
	at play.sbt.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:34)
	at play.sbt.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:34)
	at scala.Option.map(Option.scala:230)
	at play.sbt.run.PlayReload$.$anonfun$taskFailureHandler$1(PlayReload.scala:40)
	at scala.Option.map(Option.scala:230)
	at play.sbt.run.PlayReload$.taskFailureHandler(PlayReload.scala:35)
	at play.sbt.run.PlayReload$.compileFailure(PlayReload.scala:28)
	at play.sbt.run.PlayReload$.$anonfun$compile$3(PlayReload.scala:63)
	at scala.util.Either$LeftProjection.map(Either.scala:573)
	at play.sbt.run.PlayReload$.compile(PlayReload.scala:63)
	at play.sbt.run.PlayRun$.$anonfun$playRunTask$4(PlayRun.scala:78)
	at play.runsupport.Reloader.$anonfun$reload$1(Reloader.scala:496)
	at play.runsupport.Reloader$$anon$1.run(Reloader.scala:57)
	at java.security.AccessController.doPrivileged(Native Method)
	at play.runsupport.Reloader$.play$runsupport$Reloader$$withReloaderContextClassLoader(Reloader.scala:63)
	at play.runsupport.Reloader.reload(Reloader.scala:496)
	at play.core.server.DevServerStart$$anon$1.get(DevServerStart.scala:141)
	at play.core.server.AkkaHttpServer.handleRequest(AkkaHttpServer.scala:302)
	at play.core.server.AkkaHttpServer.$anonfun$createServerBinding$1(AkkaHttpServer.scala:224)
	at akka.stream.impl.fusing.MapAsync$$anon$30.onPush(Ops.scala:1284)
	at akka.stream.impl.fusing.GraphInterpreter.processPush(GraphInterpreter.scala:541)
	at akka.stream.impl.fusing.GraphInterpreter.execute(GraphInterpreter.scala:423)
	at akka.stream.impl.fusing.GraphInterpreterShell.runBatch(ActorGraphInterpreter.scala:625)
	at akka.stream.impl.fusing.GraphInterpreterShell$AsyncInput.execute(ActorGraphInterpreter.scala:502)
	at akka.stream.impl.fusing.GraphInterpreterShell.processEvent(ActorGraphInterpreter.scala:600)
	at akka.stream.impl.fusing.ActorGraphInterpreter.akka$stream$impl$fusing$ActorGraphInterpreter$$processEvent(ActorGraphInterpreter.scala:769)
	at akka.stream.impl.fusing.ActorGraphInterpreter$$anonfun$receive$1.applyOrElse(ActorGraphInterpreter.scala:784)
	at akka.actor.Actor.aroundReceive(Actor.scala:537)
	at akka.actor.Actor.aroundReceive$(Actor.scala:535)
	at akka.stream.impl.fusing.ActorGraphInterpreter.aroundReceive(ActorGraphInterpreter.scala:691)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:577)
	at akka.actor.ActorCell.invoke(ActorCell.scala:547)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:270)
	at akka.dispatch.Mailbox.run(Mailbox.scala:231)
	at akka.dispatch.Mailbox.exec(Mailbox.scala:243)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
2021-03-06 01:31:05,779 [ERROR] from play.api.http.DefaultHttpErrorHandler in play-dev-mode-akka.actor.default-dispatcher-83 - 

! @7j1o60eao - Internal server error, for (GET) [/recetas/search/usuario/id/1] ->
 
play.sbt.PlayExceptions$CompilationException: Compilation error[not found: value _ingredientes]
	at play.sbt.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:34)
	at play.sbt.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:34)
	at scala.Option.map(Option.scala:230)
	at play.sbt.run.PlayReload$.$anonfun$taskFailureHandler$1(PlayReload.scala:40)
	at scala.Option.map(Option.scala:230)
	at play.sbt.run.PlayReload$.taskFailureHandler(PlayReload.scala:35)
	at play.sbt.run.PlayReload$.compileFailure(PlayReload.scala:28)
	at play.sbt.run.PlayReload$.$anonfun$compile$3(PlayReload.scala:63)
	at scala.util.Either$LeftProjection.map(Either.scala:573)
	at play.sbt.run.PlayReload$.compile(PlayReload.scala:63)
	at play.sbt.run.PlayRun$.$anonfun$playRunTask$4(PlayRun.scala:78)
	at play.runsupport.Reloader.$anonfun$reload$1(Reloader.scala:496)
	at play.runsupport.Reloader$$anon$1.run(Reloader.scala:57)
	at java.security.AccessController.doPrivileged(Native Method)
	at play.runsupport.Reloader$.play$runsupport$Reloader$$withReloaderContextClassLoader(Reloader.scala:63)
	at play.runsupport.Reloader.reload(Reloader.scala:496)
	at play.core.server.DevServerStart$$anon$1.get(DevServerStart.scala:141)
	at play.core.server.AkkaHttpServer.handleRequest(AkkaHttpServer.scala:302)
	at play.core.server.AkkaHttpServer.$anonfun$createServerBinding$1(AkkaHttpServer.scala:224)
	at akka.stream.impl.fusing.MapAsync$$anon$30.onPush(Ops.scala:1284)
	at akka.stream.impl.fusing.GraphInterpreter.processPush(GraphInterpreter.scala:541)
	at akka.stream.impl.fusing.GraphInterpreter.execute(GraphInterpreter.scala:423)
	at akka.stream.impl.fusing.GraphInterpreterShell.runBatch(ActorGraphInterpreter.scala:625)
	at akka.stream.impl.fusing.GraphInterpreterShell$AsyncInput.execute(ActorGraphInterpreter.scala:502)
	at akka.stream.impl.fusing.GraphInterpreterShell.processEvent(ActorGraphInterpreter.scala:600)
	at akka.stream.impl.fusing.ActorGraphInterpreter.akka$stream$impl$fusing$ActorGraphInterpreter$$processEvent(ActorGraphInterpreter.scala:769)
	at akka.stream.impl.fusing.ActorGraphInterpreter$$anonfun$receive$1.applyOrElse(ActorGraphInterpreter.scala:784)
	at akka.actor.Actor.aroundReceive(Actor.scala:537)
	at akka.actor.Actor.aroundReceive$(Actor.scala:535)
	at akka.stream.impl.fusing.ActorGraphInterpreter.aroundReceive(ActorGraphInterpreter.scala:691)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:577)
	at akka.actor.ActorCell.invoke(ActorCell.scala:547)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:270)
	at akka.dispatch.Mailbox.run(Mailbox.scala:231)
	at akka.dispatch.Mailbox.exec(Mailbox.scala:243)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
2021-03-06 01:31:37,095 [ERROR] from play.api.http.DefaultHttpErrorHandler in play-dev-mode-akka.actor.default-dispatcher-82 - 

! @7j1o628j9 - Internal server error, for (GET) [/recetas/search/usuario/id/1] ->
 
play.sbt.PlayExceptions$CompilationException: Compilation error[type mismatch;
 found   : java.util.List[models.Ingrediente]
 required: Int]
	at play.sbt.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:34)
	at play.sbt.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:34)
	at scala.Option.map(Option.scala:230)
	at play.sbt.run.PlayReload$.$anonfun$taskFailureHandler$1(PlayReload.scala:40)
	at scala.Option.map(Option.scala:230)
	at play.sbt.run.PlayReload$.taskFailureHandler(PlayReload.scala:35)
	at play.sbt.run.PlayReload$.compileFailure(PlayReload.scala:28)
	at play.sbt.run.PlayReload$.$anonfun$compile$3(PlayReload.scala:63)
	at scala.util.Either$LeftProjection.map(Either.scala:573)
	at play.sbt.run.PlayReload$.compile(PlayReload.scala:63)
	at play.sbt.run.PlayRun$.$anonfun$playRunTask$4(PlayRun.scala:78)
	at play.runsupport.Reloader.$anonfun$reload$1(Reloader.scala:496)
	at play.runsupport.Reloader$$anon$1.run(Reloader.scala:57)
	at java.security.AccessController.doPrivileged(Native Method)
	at play.runsupport.Reloader$.play$runsupport$Reloader$$withReloaderContextClassLoader(Reloader.scala:63)
	at play.runsupport.Reloader.reload(Reloader.scala:496)
	at play.core.server.DevServerStart$$anon$1.get(DevServerStart.scala:141)
	at play.core.server.AkkaHttpServer.handleRequest(AkkaHttpServer.scala:302)
	at play.core.server.AkkaHttpServer.$anonfun$createServerBinding$1(AkkaHttpServer.scala:224)
	at akka.stream.impl.fusing.MapAsync$$anon$30.onPush(Ops.scala:1284)
	at akka.stream.impl.fusing.GraphInterpreter.processPush(GraphInterpreter.scala:541)
	at akka.stream.impl.fusing.GraphInterpreter.execute(GraphInterpreter.scala:423)
	at akka.stream.impl.fusing.GraphInterpreterShell.runBatch(ActorGraphInterpreter.scala:625)
	at akka.stream.impl.fusing.GraphInterpreterShell$AsyncInput.execute(ActorGraphInterpreter.scala:502)
	at akka.stream.impl.fusing.GraphInterpreterShell.processEvent(ActorGraphInterpreter.scala:600)
	at akka.stream.impl.fusing.ActorGraphInterpreter.akka$stream$impl$fusing$ActorGraphInterpreter$$processEvent(ActorGraphInterpreter.scala:769)
	at akka.stream.impl.fusing.ActorGraphInterpreter$$anonfun$receive$1.applyOrElse(ActorGraphInterpreter.scala:784)
	at akka.actor.Actor.aroundReceive(Actor.scala:537)
	at akka.actor.Actor.aroundReceive$(Actor.scala:535)
	at akka.stream.impl.fusing.ActorGraphInterpreter.aroundReceive(ActorGraphInterpreter.scala:691)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:577)
	at akka.actor.ActorCell.invoke(ActorCell.scala:547)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:270)
	at akka.dispatch.Mailbox.run(Mailbox.scala:231)
	at akka.dispatch.Mailbox.exec(Mailbox.scala:243)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
2021-03-06 01:32:54,687 [INFO] from play.api.db.HikariCPConnectionPool in application-akka.actor.internal-dispatcher-8 - Shutting down connection pool.
2021-03-06 01:32:54,889 [INFO] from play.api.db.DefaultDBApi in play-dev-mode-akka.actor.default-dispatcher-95 - Database [default] initialized
2021-03-06 01:32:54,889 [INFO] from play.api.db.HikariCPConnectionPool in play-dev-mode-akka.actor.default-dispatcher-95 - Creating Pool for datasource 'default'
2021-03-06 01:32:55,062 [INFO] from play.api.http.EnabledFilters in play-dev-mode-akka.actor.default-dispatcher-95 - Enabled Filters (see <https://www.playframework.com/documentation/latest/Filters>):

    play.filters.csrf.CSRFFilter
    play.filters.headers.SecurityHeadersFilter
    play.filters.hosts.AllowedHostsFilter

2021-03-06 01:32:55,065 [INFO] from play.api.Play in play-dev-mode-akka.actor.default-dispatcher-95 - Application started (Dev) (no global state)
2021-03-06 01:33:57,003 [ERROR] from play.api.http.DefaultHttpErrorHandler in play-dev-mode-akka.actor.default-dispatcher-95 - 

! @7j1o6a7ge - Internal server error, for (GET) [/recetas/search/usuario/id/1] ->
 
play.sbt.PlayExceptions$CompilationException: Compilation error[too many arguments (2) for method apply: (receta: models.Receta)play.twirl.api.XmlFormat.Appendable in object receta]
	at play.sbt.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:34)
	at play.sbt.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:34)
	at scala.Option.map(Option.scala:230)
	at play.sbt.run.PlayReload$.$anonfun$taskFailureHandler$1(PlayReload.scala:40)
	at scala.Option.map(Option.scala:230)
	at play.sbt.run.PlayReload$.taskFailureHandler(PlayReload.scala:35)
	at play.sbt.run.PlayReload$.compileFailure(PlayReload.scala:28)
	at play.sbt.run.PlayReload$.$anonfun$compile$3(PlayReload.scala:63)
	at scala.util.Either$LeftProjection.map(Either.scala:573)
	at play.sbt.run.PlayReload$.compile(PlayReload.scala:63)
	at play.sbt.run.PlayRun$.$anonfun$playRunTask$4(PlayRun.scala:78)
	at play.runsupport.Reloader.$anonfun$reload$1(Reloader.scala:496)
	at play.runsupport.Reloader$$anon$1.run(Reloader.scala:57)
	at java.security.AccessController.doPrivileged(Native Method)
	at play.runsupport.Reloader$.play$runsupport$Reloader$$withReloaderContextClassLoader(Reloader.scala:63)
	at play.runsupport.Reloader.reload(Reloader.scala:496)
	at play.core.server.DevServerStart$$anon$1.get(DevServerStart.scala:141)
	at play.core.server.AkkaHttpServer.handleRequest(AkkaHttpServer.scala:302)
	at play.core.server.AkkaHttpServer.$anonfun$createServerBinding$1(AkkaHttpServer.scala:224)
	at akka.stream.impl.fusing.MapAsync$$anon$30.onPush(Ops.scala:1284)
	at akka.stream.impl.fusing.GraphInterpreter.processPush(GraphInterpreter.scala:541)
	at akka.stream.impl.fusing.GraphInterpreter.execute(GraphInterpreter.scala:423)
	at akka.stream.impl.fusing.GraphInterpreterShell.runBatch(ActorGraphInterpreter.scala:625)
	at akka.stream.impl.fusing.GraphInterpreterShell$AsyncInput.execute(ActorGraphInterpreter.scala:502)
	at akka.stream.impl.fusing.GraphInterpreterShell.processEvent(ActorGraphInterpreter.scala:600)
	at akka.stream.impl.fusing.ActorGraphInterpreter.akka$stream$impl$fusing$ActorGraphInterpreter$$processEvent(ActorGraphInterpreter.scala:769)
	at akka.stream.impl.fusing.ActorGraphInterpreter$$anonfun$receive$1.applyOrElse(ActorGraphInterpreter.scala:784)
	at akka.actor.Actor.aroundReceive(Actor.scala:537)
	at akka.actor.Actor.aroundReceive$(Actor.scala:535)
	at akka.stream.impl.fusing.ActorGraphInterpreter.aroundReceive(ActorGraphInterpreter.scala:691)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:577)
	at akka.actor.ActorCell.invoke(ActorCell.scala:547)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:270)
	at akka.dispatch.Mailbox.run(Mailbox.scala:231)
	at akka.dispatch.Mailbox.exec(Mailbox.scala:243)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
2021-03-06 01:33:59,222 [ERROR] from play.api.http.DefaultHttpErrorHandler in play-dev-mode-akka.actor.default-dispatcher-83 - 

! @7j1o6aapk - Internal server error, for (GET) [/recetas/search/usuario/id/1] ->
 
play.sbt.PlayExceptions$CompilationException: Compilation error[too many arguments (2) for method apply: (receta: models.Receta)play.twirl.api.XmlFormat.Appendable in object receta]
	at play.sbt.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:34)
	at play.sbt.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:34)
	at scala.Option.map(Option.scala:230)
	at play.sbt.run.PlayReload$.$anonfun$taskFailureHandler$1(PlayReload.scala:40)
	at scala.Option.map(Option.scala:230)
	at play.sbt.run.PlayReload$.taskFailureHandler(PlayReload.scala:35)
	at play.sbt.run.PlayReload$.compileFailure(PlayReload.scala:28)
	at play.sbt.run.PlayReload$.$anonfun$compile$3(PlayReload.scala:63)
	at scala.util.Either$LeftProjection.map(Either.scala:573)
	at play.sbt.run.PlayReload$.compile(PlayReload.scala:63)
	at play.sbt.run.PlayRun$.$anonfun$playRunTask$4(PlayRun.scala:78)
	at play.runsupport.Reloader.$anonfun$reload$1(Reloader.scala:496)
	at play.runsupport.Reloader$$anon$1.run(Reloader.scala:57)
	at java.security.AccessController.doPrivileged(Native Method)
	at play.runsupport.Reloader$.play$runsupport$Reloader$$withReloaderContextClassLoader(Reloader.scala:63)
	at play.runsupport.Reloader.reload(Reloader.scala:496)
	at play.core.server.DevServerStart$$anon$1.get(DevServerStart.scala:141)
	at play.core.server.AkkaHttpServer.handleRequest(AkkaHttpServer.scala:302)
	at play.core.server.AkkaHttpServer.$anonfun$createServerBinding$1(AkkaHttpServer.scala:224)
	at akka.stream.impl.fusing.MapAsync$$anon$30.onPush(Ops.scala:1284)
	at akka.stream.impl.fusing.GraphInterpreter.processPush(GraphInterpreter.scala:541)
	at akka.stream.impl.fusing.GraphInterpreter.execute(GraphInterpreter.scala:423)
	at akka.stream.impl.fusing.GraphInterpreterShell.runBatch(ActorGraphInterpreter.scala:625)
	at akka.stream.impl.fusing.GraphInterpreterShell$AsyncInput.execute(ActorGraphInterpreter.scala:502)
	at akka.stream.impl.fusing.GraphInterpreterShell.processEvent(ActorGraphInterpreter.scala:600)
	at akka.stream.impl.fusing.ActorGraphInterpreter.akka$stream$impl$fusing$ActorGraphInterpreter$$processEvent(ActorGraphInterpreter.scala:769)
	at akka.stream.impl.fusing.ActorGraphInterpreter$$anonfun$receive$1.applyOrElse(ActorGraphInterpreter.scala:784)
	at akka.actor.Actor.aroundReceive(Actor.scala:537)
	at akka.actor.Actor.aroundReceive$(Actor.scala:535)
	at akka.stream.impl.fusing.ActorGraphInterpreter.aroundReceive(ActorGraphInterpreter.scala:691)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:577)
	at akka.actor.ActorCell.invoke(ActorCell.scala:547)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:270)
	at akka.dispatch.Mailbox.run(Mailbox.scala:231)
	at akka.dispatch.Mailbox.exec(Mailbox.scala:243)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
2021-03-06 01:34:31,483 [ERROR] from play.api.http.DefaultHttpErrorHandler in play-dev-mode-akka.actor.default-dispatcher-83 - 

! @7j1o6c6ig - Internal server error, for (GET) [/recetas/search/usuario/id/1] ->
 
play.sbt.PlayExceptions$CompilationException: Compilation error[method render in class views.xml.receta cannot be applied to given types;
  required: models.Receta
  found: models.Receta,java.util.List<models.Ingrediente>
  reason: actual and formal argument lists differ in length]
	at play.sbt.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:34)
	at play.sbt.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:34)
	at scala.Option.map(Option.scala:230)
	at play.sbt.run.PlayReload$.$anonfun$taskFailureHandler$1(PlayReload.scala:40)
	at scala.Option.map(Option.scala:230)
	at play.sbt.run.PlayReload$.taskFailureHandler(PlayReload.scala:35)
	at play.sbt.run.PlayReload$.compileFailure(PlayReload.scala:28)
	at play.sbt.run.PlayReload$.$anonfun$compile$3(PlayReload.scala:63)
	at scala.util.Either$LeftProjection.map(Either.scala:573)
	at play.sbt.run.PlayReload$.compile(PlayReload.scala:63)
	at play.sbt.run.PlayRun$.$anonfun$playRunTask$4(PlayRun.scala:78)
	at play.runsupport.Reloader.$anonfun$reload$1(Reloader.scala:496)
	at play.runsupport.Reloader$$anon$1.run(Reloader.scala:57)
	at java.security.AccessController.doPrivileged(Native Method)
	at play.runsupport.Reloader$.play$runsupport$Reloader$$withReloaderContextClassLoader(Reloader.scala:63)
	at play.runsupport.Reloader.reload(Reloader.scala:496)
	at play.core.server.DevServerStart$$anon$1.get(DevServerStart.scala:141)
	at play.core.server.AkkaHttpServer.handleRequest(AkkaHttpServer.scala:302)
	at play.core.server.AkkaHttpServer.$anonfun$createServerBinding$1(AkkaHttpServer.scala:224)
	at akka.stream.impl.fusing.MapAsync$$anon$30.onPush(Ops.scala:1284)
	at akka.stream.impl.fusing.GraphInterpreter.processPush(GraphInterpreter.scala:541)
	at akka.stream.impl.fusing.GraphInterpreter.execute(GraphInterpreter.scala:423)
	at akka.stream.impl.fusing.GraphInterpreterShell.runBatch(ActorGraphInterpreter.scala:625)
	at akka.stream.impl.fusing.GraphInterpreterShell$AsyncInput.execute(ActorGraphInterpreter.scala:502)
	at akka.stream.impl.fusing.GraphInterpreterShell.processEvent(ActorGraphInterpreter.scala:600)
	at akka.stream.impl.fusing.ActorGraphInterpreter.akka$stream$impl$fusing$ActorGraphInterpreter$$processEvent(ActorGraphInterpreter.scala:769)
	at akka.stream.impl.fusing.ActorGraphInterpreter$$anonfun$receive$1.applyOrElse(ActorGraphInterpreter.scala:784)
	at akka.actor.Actor.aroundReceive(Actor.scala:537)
	at akka.actor.Actor.aroundReceive$(Actor.scala:535)
	at akka.stream.impl.fusing.ActorGraphInterpreter.aroundReceive(ActorGraphInterpreter.scala:691)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:577)
	at akka.actor.ActorCell.invoke(ActorCell.scala:547)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:270)
	at akka.dispatch.Mailbox.run(Mailbox.scala:231)
	at akka.dispatch.Mailbox.exec(Mailbox.scala:243)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
2021-03-06 01:35:03,525 [INFO] from play.api.db.HikariCPConnectionPool in application-akka.actor.internal-dispatcher-7 - Shutting down connection pool.
2021-03-06 01:35:03,808 [INFO] from play.api.db.DefaultDBApi in play-dev-mode-akka.actor.default-dispatcher-83 - Database [default] initialized
2021-03-06 01:35:03,809 [INFO] from play.api.db.HikariCPConnectionPool in play-dev-mode-akka.actor.default-dispatcher-83 - Creating Pool for datasource 'default'
2021-03-06 01:35:03,976 [INFO] from play.api.http.EnabledFilters in play-dev-mode-akka.actor.default-dispatcher-83 - Enabled Filters (see <https://www.playframework.com/documentation/latest/Filters>):

    play.filters.csrf.CSRFFilter
    play.filters.headers.SecurityHeadersFilter
    play.filters.hosts.AllowedHostsFilter

2021-03-06 01:35:03,979 [INFO] from play.api.Play in play-dev-mode-akka.actor.default-dispatcher-83 - Application started (Dev) (no global state)
2021-03-06 01:37:11,667 [INFO] from play.api.db.HikariCPConnectionPool in application-akka.actor.internal-dispatcher-7 - Shutting down connection pool.
2021-03-06 01:37:11,896 [INFO] from play.api.db.DefaultDBApi in play-dev-mode-akka.actor.default-dispatcher-109 - Database [default] initialized
2021-03-06 01:37:11,896 [INFO] from play.api.db.HikariCPConnectionPool in play-dev-mode-akka.actor.default-dispatcher-109 - Creating Pool for datasource 'default'
2021-03-06 01:37:12,077 [INFO] from play.api.http.EnabledFilters in play-dev-mode-akka.actor.default-dispatcher-109 - Enabled Filters (see <https://www.playframework.com/documentation/latest/Filters>):

    play.filters.csrf.CSRFFilter
    play.filters.headers.SecurityHeadersFilter
    play.filters.hosts.AllowedHostsFilter

2021-03-06 01:37:12,081 [INFO] from play.api.Play in play-dev-mode-akka.actor.default-dispatcher-109 - Application started (Dev) (no global state)
2021-03-06 02:44:20,593 [INFO] from play.api.db.HikariCPConnectionPool in application-akka.actor.internal-dispatcher-9 - Shutting down connection pool.
2021-03-06 02:44:20,816 [INFO] from play.api.db.DefaultDBApi in play-dev-mode-akka.actor.default-dispatcher-121 - Database [default] initialized
2021-03-06 02:44:20,816 [INFO] from play.api.db.HikariCPConnectionPool in play-dev-mode-akka.actor.default-dispatcher-121 - Creating Pool for datasource 'default'
2021-03-06 02:44:21,010 [INFO] from play.api.http.EnabledFilters in play-dev-mode-akka.actor.default-dispatcher-121 - Enabled Filters (see <https://www.playframework.com/documentation/latest/Filters>):

    play.filters.csrf.CSRFFilter
    play.filters.headers.SecurityHeadersFilter
    play.filters.hosts.AllowedHostsFilter

2021-03-06 02:44:21,014 [INFO] from play.api.Play in play-dev-mode-akka.actor.default-dispatcher-121 - Application started (Dev) (no global state)
2021-03-06 02:45:41,844 [ERROR] from play.api.http.DefaultHttpErrorHandler in application-akka.actor.default-dispatcher-7 - 

! @7j1ofl5l7 - Internal server error, for (PUT) [/recetas/] ->
 
play.api.http.HttpErrorHandlerExceptions$$anon$1: Execution exception[[NullPointerException: The id is null]]
	at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:359)
	at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:261)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:430)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:422)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:453)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:56)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:93)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:94)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:93)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:48)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:48)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
Caused by: java.lang.NullPointerException: The id is null
	at io.ebeaninternal.server.core.DefaultServer.find(DefaultServer.java:1113)
	at io.ebeaninternal.server.core.DefaultServer.find(DefaultServer.java:1103)
	at io.ebean.Finder.byId(Finder.java:148)
	at models.Descripcion.findById(Descripcion.java:62)
	at controllers.RecetasController.updateReceta(RecetasController.java:351)
	at router.Routes$$anonfun$routes$1.$anonfun$applyOrElse$39(Routes.scala:527)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:150)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:149)
	at play.core.routing.HandlerInvokerFactory$JavaActionInvokerFactory$$anon$3$$anon$4$$anon$5.invocation(HandlerInvoker.scala:115)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:119)
	at play.http.DefaultActionCreator$1.call(DefaultActionCreator.java:33)
	at play.core.j.JavaAction.$anonfun$apply$8(JavaAction.scala:175)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:661)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:430)
	at play.core.j.HttpExecutionContext.$anonfun$execute$1(HttpExecutionContext.scala:64)
	at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:70)
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:59)
	at scala.concurrent.impl.Promise$Transformation.submitWithValue(Promise.scala:392)
	at scala.concurrent.impl.Promise$DefaultPromise.submitWithValue(Promise.scala:302)
	at scala.concurrent.impl.Promise$DefaultPromise.dispatchOrAddCallbacks(Promise.scala:276)
	at scala.concurrent.impl.Promise$DefaultPromise.map(Promise.scala:146)
	at scala.concurrent.Future$.apply(Future.scala:661)
	at play.core.j.JavaAction.apply(JavaAction.scala:176)
	at play.api.mvc.Action.$anonfun$apply$4(Action.scala:82)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:433)
	... 11 common frames omitted
2021-03-06 02:50:30,482 [INFO] from play.api.db.HikariCPConnectionPool in application-akka.actor.internal-dispatcher-8 - Shutting down connection pool.
2021-03-06 02:50:30,696 [INFO] from play.api.db.DefaultDBApi in play-dev-mode-akka.actor.default-dispatcher-131 - Database [default] initialized
2021-03-06 02:50:30,697 [INFO] from play.api.db.HikariCPConnectionPool in play-dev-mode-akka.actor.default-dispatcher-131 - Creating Pool for datasource 'default'
2021-03-06 02:50:30,874 [INFO] from play.api.http.EnabledFilters in play-dev-mode-akka.actor.default-dispatcher-131 - Enabled Filters (see <https://www.playframework.com/documentation/latest/Filters>):

    play.filters.csrf.CSRFFilter
    play.filters.headers.SecurityHeadersFilter
    play.filters.hosts.AllowedHostsFilter

2021-03-06 02:50:30,878 [INFO] from play.api.Play in play-dev-mode-akka.actor.default-dispatcher-131 - Application started (Dev) (no global state)
2021-03-06 02:50:30,960 [ERROR] from play.api.http.DefaultHttpErrorHandler in application-akka.actor.default-dispatcher-6 - 

! @7j1ogbhd3 - Internal server error, for (PUT) [/recetas/] ->
 
play.api.http.HttpErrorHandlerExceptions$$anon$1: Execution exception[[PersistenceException: ManyToMany bean models.Receta@1e7eb0cd does not have an Id value.]]
	at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:359)
	at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:261)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:430)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:422)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:453)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:56)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:93)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:94)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:93)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:48)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:48)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
Caused by: javax.persistence.PersistenceException: ManyToMany bean models.Receta@1e7eb0cd does not have an Id value.
	at io.ebeaninternal.server.persist.SaveManyBeans.saveAssocManyIntersection(SaveManyBeans.java:307)
	at io.ebeaninternal.server.persist.SaveManyBeans.save(SaveManyBeans.java:69)
	at io.ebeaninternal.server.persist.DefaultPersister.saveMany(DefaultPersister.java:946)
	at io.ebeaninternal.server.persist.DefaultPersister.saveAssocMany(DefaultPersister.java:937)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:511)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:455)
	at io.ebeaninternal.server.persist.DefaultPersister.save(DefaultPersister.java:438)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1745)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1737)
	at io.ebean.Model.save(Model.java:188)
	at controllers.RecetasController.updateReceta(RecetasController.java:368)
	at router.Routes$$anonfun$routes$1.$anonfun$applyOrElse$39(Routes.scala:527)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:150)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:149)
	at play.core.routing.HandlerInvokerFactory$JavaActionInvokerFactory$$anon$3$$anon$4$$anon$5.invocation(HandlerInvoker.scala:115)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:119)
	at play.http.DefaultActionCreator$1.call(DefaultActionCreator.java:33)
	at play.core.j.JavaAction.$anonfun$apply$8(JavaAction.scala:175)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:661)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:430)
	at play.core.j.HttpExecutionContext.$anonfun$execute$1(HttpExecutionContext.scala:64)
	at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:70)
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:59)
	at scala.concurrent.impl.Promise$Transformation.submitWithValue(Promise.scala:392)
	at scala.concurrent.impl.Promise$DefaultPromise.submitWithValue(Promise.scala:302)
	at scala.concurrent.impl.Promise$DefaultPromise.dispatchOrAddCallbacks(Promise.scala:276)
	at scala.concurrent.impl.Promise$DefaultPromise.map(Promise.scala:146)
	at scala.concurrent.Future$.apply(Future.scala:661)
	at play.core.j.JavaAction.apply(JavaAction.scala:176)
	at play.api.mvc.Action.$anonfun$apply$4(Action.scala:82)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:433)
	... 11 common frames omitted
2021-03-06 02:56:52,531 [ERROR] from play.api.http.DefaultHttpErrorHandler in application-akka.actor.default-dispatcher-7 - 

! @7j1oh79oo - Internal server error, for (PUT) [/recetas/] ->
 
play.api.http.HttpErrorHandlerExceptions$$anon$1: Execution exception[[PersistenceException: ManyToMany bean models.Receta@3cb24c7c does not have an Id value.]]
	at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:359)
	at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:261)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:430)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:422)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:453)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:56)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:93)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:94)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:93)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:48)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:48)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
Caused by: javax.persistence.PersistenceException: ManyToMany bean models.Receta@3cb24c7c does not have an Id value.
	at io.ebeaninternal.server.persist.SaveManyBeans.saveAssocManyIntersection(SaveManyBeans.java:307)
	at io.ebeaninternal.server.persist.SaveManyBeans.save(SaveManyBeans.java:69)
	at io.ebeaninternal.server.persist.DefaultPersister.saveMany(DefaultPersister.java:946)
	at io.ebeaninternal.server.persist.DefaultPersister.saveAssocMany(DefaultPersister.java:937)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:511)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:455)
	at io.ebeaninternal.server.persist.DefaultPersister.save(DefaultPersister.java:438)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1745)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1737)
	at io.ebean.Model.save(Model.java:188)
	at controllers.RecetasController.updateReceta(RecetasController.java:368)
	at router.Routes$$anonfun$routes$1.$anonfun$applyOrElse$39(Routes.scala:527)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:150)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:149)
	at play.core.routing.HandlerInvokerFactory$JavaActionInvokerFactory$$anon$3$$anon$4$$anon$5.invocation(HandlerInvoker.scala:115)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:119)
	at play.http.DefaultActionCreator$1.call(DefaultActionCreator.java:33)
	at play.core.j.JavaAction.$anonfun$apply$8(JavaAction.scala:175)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:661)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:430)
	at play.core.j.HttpExecutionContext.$anonfun$execute$1(HttpExecutionContext.scala:64)
	at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:70)
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:59)
	at scala.concurrent.impl.Promise$Transformation.submitWithValue(Promise.scala:392)
	at scala.concurrent.impl.Promise$DefaultPromise.submitWithValue(Promise.scala:302)
	at scala.concurrent.impl.Promise$DefaultPromise.dispatchOrAddCallbacks(Promise.scala:276)
	at scala.concurrent.impl.Promise$DefaultPromise.map(Promise.scala:146)
	at scala.concurrent.Future$.apply(Future.scala:661)
	at play.core.j.JavaAction.apply(JavaAction.scala:176)
	at play.api.mvc.Action.$anonfun$apply$4(Action.scala:82)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:433)
	... 11 common frames omitted
2021-03-06 03:00:16,047 [ERROR] from play.api.http.DefaultHttpErrorHandler in application-akka.actor.default-dispatcher-8 - 

! @7j1ohip0b - Internal server error, for (PUT) [/recetas/] ->
 
play.api.http.HttpErrorHandlerExceptions$$anon$1: Execution exception[[PersistenceException: ManyToMany bean models.Receta@2aba020e does not have an Id value.]]
	at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:359)
	at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:261)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:430)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:422)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:453)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:56)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:93)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:94)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:93)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:48)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:48)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
Caused by: javax.persistence.PersistenceException: ManyToMany bean models.Receta@2aba020e does not have an Id value.
	at io.ebeaninternal.server.persist.SaveManyBeans.saveAssocManyIntersection(SaveManyBeans.java:307)
	at io.ebeaninternal.server.persist.SaveManyBeans.save(SaveManyBeans.java:69)
	at io.ebeaninternal.server.persist.DefaultPersister.saveMany(DefaultPersister.java:946)
	at io.ebeaninternal.server.persist.DefaultPersister.saveAssocMany(DefaultPersister.java:937)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:511)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:455)
	at io.ebeaninternal.server.persist.DefaultPersister.save(DefaultPersister.java:438)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1745)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1737)
	at io.ebean.Model.save(Model.java:188)
	at controllers.RecetasController.updateReceta(RecetasController.java:368)
	at router.Routes$$anonfun$routes$1.$anonfun$applyOrElse$39(Routes.scala:527)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:150)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:149)
	at play.core.routing.HandlerInvokerFactory$JavaActionInvokerFactory$$anon$3$$anon$4$$anon$5.invocation(HandlerInvoker.scala:115)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:119)
	at play.http.DefaultActionCreator$1.call(DefaultActionCreator.java:33)
	at play.core.j.JavaAction.$anonfun$apply$8(JavaAction.scala:175)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:661)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:430)
	at play.core.j.HttpExecutionContext.$anonfun$execute$1(HttpExecutionContext.scala:64)
	at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:70)
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:59)
	at scala.concurrent.impl.Promise$Transformation.submitWithValue(Promise.scala:392)
	at scala.concurrent.impl.Promise$DefaultPromise.submitWithValue(Promise.scala:302)
	at scala.concurrent.impl.Promise$DefaultPromise.dispatchOrAddCallbacks(Promise.scala:276)
	at scala.concurrent.impl.Promise$DefaultPromise.map(Promise.scala:146)
	at scala.concurrent.Future$.apply(Future.scala:661)
	at play.core.j.JavaAction.apply(JavaAction.scala:176)
	at play.api.mvc.Action.$anonfun$apply$4(Action.scala:82)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:433)
	... 11 common frames omitted
2021-03-06 03:00:51,227 [ERROR] from play.api.http.DefaultHttpErrorHandler in application-akka.actor.default-dispatcher-9 - 

! @7j1ohkp1e - Internal server error, for (PUT) [/recetas/] ->
 
play.api.http.HttpErrorHandlerExceptions$$anon$1: Execution exception[[PersistenceException: ManyToMany bean models.Receta@64647bce does not have an Id value.]]
	at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:359)
	at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:261)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:430)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:422)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:453)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:56)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:93)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:94)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:93)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:48)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:48)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
Caused by: javax.persistence.PersistenceException: ManyToMany bean models.Receta@64647bce does not have an Id value.
	at io.ebeaninternal.server.persist.SaveManyBeans.saveAssocManyIntersection(SaveManyBeans.java:307)
	at io.ebeaninternal.server.persist.SaveManyBeans.save(SaveManyBeans.java:69)
	at io.ebeaninternal.server.persist.DefaultPersister.saveMany(DefaultPersister.java:946)
	at io.ebeaninternal.server.persist.DefaultPersister.saveAssocMany(DefaultPersister.java:937)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:511)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:455)
	at io.ebeaninternal.server.persist.DefaultPersister.save(DefaultPersister.java:438)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1745)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1737)
	at io.ebean.Model.save(Model.java:188)
	at controllers.RecetasController.updateReceta(RecetasController.java:368)
	at router.Routes$$anonfun$routes$1.$anonfun$applyOrElse$39(Routes.scala:527)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:150)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:149)
	at play.core.routing.HandlerInvokerFactory$JavaActionInvokerFactory$$anon$3$$anon$4$$anon$5.invocation(HandlerInvoker.scala:115)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:119)
	at play.http.DefaultActionCreator$1.call(DefaultActionCreator.java:33)
	at play.core.j.JavaAction.$anonfun$apply$8(JavaAction.scala:175)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:661)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:430)
	at play.core.j.HttpExecutionContext.$anonfun$execute$1(HttpExecutionContext.scala:64)
	at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:70)
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:59)
	at scala.concurrent.impl.Promise$Transformation.submitWithValue(Promise.scala:392)
	at scala.concurrent.impl.Promise$DefaultPromise.submitWithValue(Promise.scala:302)
	at scala.concurrent.impl.Promise$DefaultPromise.dispatchOrAddCallbacks(Promise.scala:276)
	at scala.concurrent.impl.Promise$DefaultPromise.map(Promise.scala:146)
	at scala.concurrent.Future$.apply(Future.scala:661)
	at play.core.j.JavaAction.apply(JavaAction.scala:176)
	at play.api.mvc.Action.$anonfun$apply$4(Action.scala:82)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:433)
	... 11 common frames omitted
2021-03-06 03:00:52,369 [ERROR] from play.api.http.DefaultHttpErrorHandler in application-akka.actor.default-dispatcher-9 - 

! @7j1ohl0jc - Internal server error, for (PUT) [/recetas/] ->
 
play.api.http.HttpErrorHandlerExceptions$$anon$1: Execution exception[[PersistenceException: ManyToMany bean models.Receta@1b62a870 does not have an Id value.]]
	at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:359)
	at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:261)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:430)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:422)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:453)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:56)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:93)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:94)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:93)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:48)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:48)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
Caused by: javax.persistence.PersistenceException: ManyToMany bean models.Receta@1b62a870 does not have an Id value.
	at io.ebeaninternal.server.persist.SaveManyBeans.saveAssocManyIntersection(SaveManyBeans.java:307)
	at io.ebeaninternal.server.persist.SaveManyBeans.save(SaveManyBeans.java:69)
	at io.ebeaninternal.server.persist.DefaultPersister.saveMany(DefaultPersister.java:946)
	at io.ebeaninternal.server.persist.DefaultPersister.saveAssocMany(DefaultPersister.java:937)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:511)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:455)
	at io.ebeaninternal.server.persist.DefaultPersister.save(DefaultPersister.java:438)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1745)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1737)
	at io.ebean.Model.save(Model.java:188)
	at controllers.RecetasController.updateReceta(RecetasController.java:368)
	at router.Routes$$anonfun$routes$1.$anonfun$applyOrElse$39(Routes.scala:527)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:150)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:149)
	at play.core.routing.HandlerInvokerFactory$JavaActionInvokerFactory$$anon$3$$anon$4$$anon$5.invocation(HandlerInvoker.scala:115)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:119)
	at play.http.DefaultActionCreator$1.call(DefaultActionCreator.java:33)
	at play.core.j.JavaAction.$anonfun$apply$8(JavaAction.scala:175)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:661)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:430)
	at play.core.j.HttpExecutionContext.$anonfun$execute$1(HttpExecutionContext.scala:64)
	at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:70)
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:59)
	at scala.concurrent.impl.Promise$Transformation.submitWithValue(Promise.scala:392)
	at scala.concurrent.impl.Promise$DefaultPromise.submitWithValue(Promise.scala:302)
	at scala.concurrent.impl.Promise$DefaultPromise.dispatchOrAddCallbacks(Promise.scala:276)
	at scala.concurrent.impl.Promise$DefaultPromise.map(Promise.scala:146)
	at scala.concurrent.Future$.apply(Future.scala:661)
	at play.core.j.JavaAction.apply(JavaAction.scala:176)
	at play.api.mvc.Action.$anonfun$apply$4(Action.scala:82)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:433)
	... 11 common frames omitted
2021-03-06 03:07:24,650 [INFO] from play.api.db.HikariCPConnectionPool in application-akka.actor.internal-dispatcher-10 - Shutting down connection pool.
2021-03-06 03:07:24,871 [INFO] from play.api.db.DefaultDBApi in play-dev-mode-akka.actor.default-dispatcher-159 - Database [default] initialized
2021-03-06 03:07:24,871 [INFO] from play.api.db.HikariCPConnectionPool in play-dev-mode-akka.actor.default-dispatcher-159 - Creating Pool for datasource 'default'
2021-03-06 03:07:25,042 [INFO] from play.api.http.EnabledFilters in play-dev-mode-akka.actor.default-dispatcher-159 - Enabled Filters (see <https://www.playframework.com/documentation/latest/Filters>):

    play.filters.csrf.CSRFFilter
    play.filters.headers.SecurityHeadersFilter
    play.filters.hosts.AllowedHostsFilter

2021-03-06 03:07:25,046 [INFO] from play.api.Play in play-dev-mode-akka.actor.default-dispatcher-159 - Application started (Dev) (no global state)
2021-03-06 03:08:03,609 [ERROR] from play.api.http.DefaultHttpErrorHandler in application-akka.actor.default-dispatcher-7 - 

! @7j1oijehg - Internal server error, for (PUT) [/recetas/] ->
 
play.api.http.HttpErrorHandlerExceptions$$anon$1: Execution exception[[PersistenceException: ManyToMany bean models.Receta@8e0172d does not have an Id value.]]
	at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:359)
	at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:261)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:430)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:422)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:453)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:56)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:93)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:94)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:93)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:48)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:48)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
Caused by: javax.persistence.PersistenceException: ManyToMany bean models.Receta@8e0172d does not have an Id value.
	at io.ebeaninternal.server.persist.SaveManyBeans.saveAssocManyIntersection(SaveManyBeans.java:307)
	at io.ebeaninternal.server.persist.SaveManyBeans.save(SaveManyBeans.java:69)
	at io.ebeaninternal.server.persist.DefaultPersister.saveMany(DefaultPersister.java:946)
	at io.ebeaninternal.server.persist.DefaultPersister.saveAssocMany(DefaultPersister.java:937)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:511)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:455)
	at io.ebeaninternal.server.persist.DefaultPersister.save(DefaultPersister.java:438)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1745)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1737)
	at io.ebean.Model.save(Model.java:188)
	at controllers.RecetasController.updateReceta(RecetasController.java:367)
	at router.Routes$$anonfun$routes$1.$anonfun$applyOrElse$39(Routes.scala:527)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:150)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:149)
	at play.core.routing.HandlerInvokerFactory$JavaActionInvokerFactory$$anon$3$$anon$4$$anon$5.invocation(HandlerInvoker.scala:115)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:119)
	at play.http.DefaultActionCreator$1.call(DefaultActionCreator.java:33)
	at play.core.j.JavaAction.$anonfun$apply$8(JavaAction.scala:175)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:661)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:430)
	at play.core.j.HttpExecutionContext.$anonfun$execute$1(HttpExecutionContext.scala:64)
	at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:70)
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:59)
	at scala.concurrent.impl.Promise$Transformation.submitWithValue(Promise.scala:392)
	at scala.concurrent.impl.Promise$DefaultPromise.submitWithValue(Promise.scala:302)
	at scala.concurrent.impl.Promise$DefaultPromise.dispatchOrAddCallbacks(Promise.scala:276)
	at scala.concurrent.impl.Promise$DefaultPromise.map(Promise.scala:146)
	at scala.concurrent.Future$.apply(Future.scala:661)
	at play.core.j.JavaAction.apply(JavaAction.scala:176)
	at play.api.mvc.Action.$anonfun$apply$4(Action.scala:82)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:433)
	... 11 common frames omitted
2021-03-06 03:12:54,387 [INFO] from play.api.db.HikariCPConnectionPool in application-akka.actor.internal-dispatcher-8 - Shutting down connection pool.
2021-03-06 03:12:54,571 [INFO] from play.api.db.DefaultDBApi in play-dev-mode-akka.actor.default-dispatcher-171 - Database [default] initialized
2021-03-06 03:12:54,571 [INFO] from play.api.db.HikariCPConnectionPool in play-dev-mode-akka.actor.default-dispatcher-171 - Creating Pool for datasource 'default'
2021-03-06 03:12:54,761 [INFO] from play.api.http.EnabledFilters in play-dev-mode-akka.actor.default-dispatcher-171 - Enabled Filters (see <https://www.playframework.com/documentation/latest/Filters>):

    play.filters.csrf.CSRFFilter
    play.filters.headers.SecurityHeadersFilter
    play.filters.hosts.AllowedHostsFilter

2021-03-06 03:12:54,764 [INFO] from play.api.Play in play-dev-mode-akka.actor.default-dispatcher-171 - Application started (Dev) (no global state)
2021-03-06 03:12:54,817 [ERROR] from play.api.http.DefaultHttpErrorHandler in application-akka.actor.default-dispatcher-6 - 

! @7j1oja3bo - Internal server error, for (PUT) [/recetas/] ->
 
play.api.http.HttpErrorHandlerExceptions$$anon$1: Execution exception[[PersistenceException: ManyToMany bean models.Receta@3fe81adb does not have an Id value.]]
	at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:359)
	at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:261)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:430)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:422)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:453)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:56)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:93)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:94)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:93)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:48)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:48)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
Caused by: javax.persistence.PersistenceException: ManyToMany bean models.Receta@3fe81adb does not have an Id value.
	at io.ebeaninternal.server.persist.SaveManyBeans.saveAssocManyIntersection(SaveManyBeans.java:307)
	at io.ebeaninternal.server.persist.SaveManyBeans.save(SaveManyBeans.java:69)
	at io.ebeaninternal.server.persist.DefaultPersister.saveMany(DefaultPersister.java:946)
	at io.ebeaninternal.server.persist.DefaultPersister.saveAssocMany(DefaultPersister.java:937)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:511)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:455)
	at io.ebeaninternal.server.persist.DefaultPersister.save(DefaultPersister.java:438)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1745)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1737)
	at io.ebean.Model.save(Model.java:188)
	at controllers.RecetasController.updateReceta(RecetasController.java:367)
	at router.Routes$$anonfun$routes$1.$anonfun$applyOrElse$39(Routes.scala:527)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:150)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:149)
	at play.core.routing.HandlerInvokerFactory$JavaActionInvokerFactory$$anon$3$$anon$4$$anon$5.invocation(HandlerInvoker.scala:115)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:119)
	at play.http.DefaultActionCreator$1.call(DefaultActionCreator.java:33)
	at play.core.j.JavaAction.$anonfun$apply$8(JavaAction.scala:175)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:661)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:430)
	at play.core.j.HttpExecutionContext.$anonfun$execute$1(HttpExecutionContext.scala:64)
	at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:70)
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:59)
	at scala.concurrent.impl.Promise$Transformation.submitWithValue(Promise.scala:392)
	at scala.concurrent.impl.Promise$DefaultPromise.submitWithValue(Promise.scala:302)
	at scala.concurrent.impl.Promise$DefaultPromise.dispatchOrAddCallbacks(Promise.scala:276)
	at scala.concurrent.impl.Promise$DefaultPromise.map(Promise.scala:146)
	at scala.concurrent.Future$.apply(Future.scala:661)
	at play.core.j.JavaAction.apply(JavaAction.scala:176)
	at play.api.mvc.Action.$anonfun$apply$4(Action.scala:82)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:433)
	... 11 common frames omitted
2021-03-06 03:16:45,215 [INFO] from play.api.db.HikariCPConnectionPool in application-akka.actor.internal-dispatcher-7 - Shutting down connection pool.
2021-03-06 03:16:45,450 [INFO] from play.api.db.DefaultDBApi in play-dev-mode-akka.actor.default-dispatcher-181 - Database [default] initialized
2021-03-06 03:16:45,451 [INFO] from play.api.db.HikariCPConnectionPool in play-dev-mode-akka.actor.default-dispatcher-181 - Creating Pool for datasource 'default'
2021-03-06 03:16:45,665 [INFO] from play.api.http.EnabledFilters in play-dev-mode-akka.actor.default-dispatcher-181 - Enabled Filters (see <https://www.playframework.com/documentation/latest/Filters>):

    play.filters.csrf.CSRFFilter
    play.filters.headers.SecurityHeadersFilter
    play.filters.hosts.AllowedHostsFilter

2021-03-06 03:16:45,670 [INFO] from play.api.Play in play-dev-mode-akka.actor.default-dispatcher-181 - Application started (Dev) (no global state)
2021-03-06 03:16:45,741 [ERROR] from play.api.http.DefaultHttpErrorHandler in application-akka.actor.default-dispatcher-6 - 

! @7j1ojn71g - Internal server error, for (PUT) [/recetas/] ->
 
play.api.http.HttpErrorHandlerExceptions$$anon$1: Execution exception[[PersistenceException: ManyToMany bean models.Receta@4e480eec does not have an Id value.]]
	at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:359)
	at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:261)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:430)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:422)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:453)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:56)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:93)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:94)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:93)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:48)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:48)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
Caused by: javax.persistence.PersistenceException: ManyToMany bean models.Receta@4e480eec does not have an Id value.
	at io.ebeaninternal.server.persist.SaveManyBeans.saveAssocManyIntersection(SaveManyBeans.java:307)
	at io.ebeaninternal.server.persist.SaveManyBeans.save(SaveManyBeans.java:69)
	at io.ebeaninternal.server.persist.DefaultPersister.saveMany(DefaultPersister.java:946)
	at io.ebeaninternal.server.persist.DefaultPersister.saveAssocMany(DefaultPersister.java:937)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:511)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:455)
	at io.ebeaninternal.server.persist.DefaultPersister.save(DefaultPersister.java:438)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1745)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1737)
	at io.ebean.Model.save(Model.java:188)
	at controllers.RecetasController.updateReceta(RecetasController.java:367)
	at router.Routes$$anonfun$routes$1.$anonfun$applyOrElse$39(Routes.scala:527)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:150)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:149)
	at play.core.routing.HandlerInvokerFactory$JavaActionInvokerFactory$$anon$3$$anon$4$$anon$5.invocation(HandlerInvoker.scala:115)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:119)
	at play.http.DefaultActionCreator$1.call(DefaultActionCreator.java:33)
	at play.core.j.JavaAction.$anonfun$apply$8(JavaAction.scala:175)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:661)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:430)
	at play.core.j.HttpExecutionContext.$anonfun$execute$1(HttpExecutionContext.scala:64)
	at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:70)
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:59)
	at scala.concurrent.impl.Promise$Transformation.submitWithValue(Promise.scala:392)
	at scala.concurrent.impl.Promise$DefaultPromise.submitWithValue(Promise.scala:302)
	at scala.concurrent.impl.Promise$DefaultPromise.dispatchOrAddCallbacks(Promise.scala:276)
	at scala.concurrent.impl.Promise$DefaultPromise.map(Promise.scala:146)
	at scala.concurrent.Future$.apply(Future.scala:661)
	at play.core.j.JavaAction.apply(JavaAction.scala:176)
	at play.api.mvc.Action.$anonfun$apply$4(Action.scala:82)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:433)
	... 11 common frames omitted
2021-03-06 03:16:48,319 [ERROR] from play.api.http.DefaultHttpErrorHandler in application-akka.actor.default-dispatcher-6 - 

! @7j1ojnamk - Internal server error, for (PUT) [/recetas/] ->
 
play.api.http.HttpErrorHandlerExceptions$$anon$1: Execution exception[[PersistenceException: ManyToMany bean models.Receta@72a198e0 does not have an Id value.]]
	at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:359)
	at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:261)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:430)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:422)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:453)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:56)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:93)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:94)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:93)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:48)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:48)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
Caused by: javax.persistence.PersistenceException: ManyToMany bean models.Receta@72a198e0 does not have an Id value.
	at io.ebeaninternal.server.persist.SaveManyBeans.saveAssocManyIntersection(SaveManyBeans.java:307)
	at io.ebeaninternal.server.persist.SaveManyBeans.save(SaveManyBeans.java:69)
	at io.ebeaninternal.server.persist.DefaultPersister.saveMany(DefaultPersister.java:946)
	at io.ebeaninternal.server.persist.DefaultPersister.saveAssocMany(DefaultPersister.java:937)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:511)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:455)
	at io.ebeaninternal.server.persist.DefaultPersister.save(DefaultPersister.java:438)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1745)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1737)
	at io.ebean.Model.save(Model.java:188)
	at controllers.RecetasController.updateReceta(RecetasController.java:367)
	at router.Routes$$anonfun$routes$1.$anonfun$applyOrElse$39(Routes.scala:527)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:150)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:149)
	at play.core.routing.HandlerInvokerFactory$JavaActionInvokerFactory$$anon$3$$anon$4$$anon$5.invocation(HandlerInvoker.scala:115)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:119)
	at play.http.DefaultActionCreator$1.call(DefaultActionCreator.java:33)
	at play.core.j.JavaAction.$anonfun$apply$8(JavaAction.scala:175)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:661)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:430)
	at play.core.j.HttpExecutionContext.$anonfun$execute$1(HttpExecutionContext.scala:64)
	at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:70)
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:59)
	at scala.concurrent.impl.Promise$Transformation.submitWithValue(Promise.scala:392)
	at scala.concurrent.impl.Promise$DefaultPromise.submitWithValue(Promise.scala:302)
	at scala.concurrent.impl.Promise$DefaultPromise.dispatchOrAddCallbacks(Promise.scala:276)
	at scala.concurrent.impl.Promise$DefaultPromise.map(Promise.scala:146)
	at scala.concurrent.Future$.apply(Future.scala:661)
	at play.core.j.JavaAction.apply(JavaAction.scala:176)
	at play.api.mvc.Action.$anonfun$apply$4(Action.scala:82)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:433)
	... 11 common frames omitted
2021-03-06 03:16:49,548 [ERROR] from play.api.http.DefaultHttpErrorHandler in application-akka.actor.default-dispatcher-6 - 

! @7j1ojnci1 - Internal server error, for (PUT) [/recetas/] ->
 
play.api.http.HttpErrorHandlerExceptions$$anon$1: Execution exception[[PersistenceException: ManyToMany bean models.Receta@3f76b776 does not have an Id value.]]
	at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:359)
	at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:261)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:430)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:422)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:453)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:56)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:93)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:94)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:93)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:48)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:48)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
Caused by: javax.persistence.PersistenceException: ManyToMany bean models.Receta@3f76b776 does not have an Id value.
	at io.ebeaninternal.server.persist.SaveManyBeans.saveAssocManyIntersection(SaveManyBeans.java:307)
	at io.ebeaninternal.server.persist.SaveManyBeans.save(SaveManyBeans.java:69)
	at io.ebeaninternal.server.persist.DefaultPersister.saveMany(DefaultPersister.java:946)
	at io.ebeaninternal.server.persist.DefaultPersister.saveAssocMany(DefaultPersister.java:937)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:511)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:455)
	at io.ebeaninternal.server.persist.DefaultPersister.save(DefaultPersister.java:438)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1745)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1737)
	at io.ebean.Model.save(Model.java:188)
	at controllers.RecetasController.updateReceta(RecetasController.java:367)
	at router.Routes$$anonfun$routes$1.$anonfun$applyOrElse$39(Routes.scala:527)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:150)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:149)
	at play.core.routing.HandlerInvokerFactory$JavaActionInvokerFactory$$anon$3$$anon$4$$anon$5.invocation(HandlerInvoker.scala:115)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:119)
	at play.http.DefaultActionCreator$1.call(DefaultActionCreator.java:33)
	at play.core.j.JavaAction.$anonfun$apply$8(JavaAction.scala:175)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:661)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:430)
	at play.core.j.HttpExecutionContext.$anonfun$execute$1(HttpExecutionContext.scala:64)
	at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:70)
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:59)
	at scala.concurrent.impl.Promise$Transformation.submitWithValue(Promise.scala:392)
	at scala.concurrent.impl.Promise$DefaultPromise.submitWithValue(Promise.scala:302)
	at scala.concurrent.impl.Promise$DefaultPromise.dispatchOrAddCallbacks(Promise.scala:276)
	at scala.concurrent.impl.Promise$DefaultPromise.map(Promise.scala:146)
	at scala.concurrent.Future$.apply(Future.scala:661)
	at play.core.j.JavaAction.apply(JavaAction.scala:176)
	at play.api.mvc.Action.$anonfun$apply$4(Action.scala:82)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:433)
	... 11 common frames omitted
2021-03-06 03:16:50,681 [ERROR] from play.api.http.DefaultHttpErrorHandler in application-akka.actor.default-dispatcher-6 - 

! @7j1ojne9g - Internal server error, for (PUT) [/recetas/] ->
 
play.api.http.HttpErrorHandlerExceptions$$anon$1: Execution exception[[PersistenceException: ManyToMany bean models.Receta@372516c6 does not have an Id value.]]
	at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:359)
	at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:261)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:430)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:422)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:453)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:56)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:93)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:94)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:93)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:48)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:48)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
Caused by: javax.persistence.PersistenceException: ManyToMany bean models.Receta@372516c6 does not have an Id value.
	at io.ebeaninternal.server.persist.SaveManyBeans.saveAssocManyIntersection(SaveManyBeans.java:307)
	at io.ebeaninternal.server.persist.SaveManyBeans.save(SaveManyBeans.java:69)
	at io.ebeaninternal.server.persist.DefaultPersister.saveMany(DefaultPersister.java:946)
	at io.ebeaninternal.server.persist.DefaultPersister.saveAssocMany(DefaultPersister.java:937)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:511)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:455)
	at io.ebeaninternal.server.persist.DefaultPersister.save(DefaultPersister.java:438)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1745)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1737)
	at io.ebean.Model.save(Model.java:188)
	at controllers.RecetasController.updateReceta(RecetasController.java:367)
	at router.Routes$$anonfun$routes$1.$anonfun$applyOrElse$39(Routes.scala:527)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:150)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:149)
	at play.core.routing.HandlerInvokerFactory$JavaActionInvokerFactory$$anon$3$$anon$4$$anon$5.invocation(HandlerInvoker.scala:115)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:119)
	at play.http.DefaultActionCreator$1.call(DefaultActionCreator.java:33)
	at play.core.j.JavaAction.$anonfun$apply$8(JavaAction.scala:175)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:661)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:430)
	at play.core.j.HttpExecutionContext.$anonfun$execute$1(HttpExecutionContext.scala:64)
	at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:70)
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:59)
	at scala.concurrent.impl.Promise$Transformation.submitWithValue(Promise.scala:392)
	at scala.concurrent.impl.Promise$DefaultPromise.submitWithValue(Promise.scala:302)
	at scala.concurrent.impl.Promise$DefaultPromise.dispatchOrAddCallbacks(Promise.scala:276)
	at scala.concurrent.impl.Promise$DefaultPromise.map(Promise.scala:146)
	at scala.concurrent.Future$.apply(Future.scala:661)
	at play.core.j.JavaAction.apply(JavaAction.scala:176)
	at play.api.mvc.Action.$anonfun$apply$4(Action.scala:82)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:433)
	... 11 common frames omitted
2021-03-06 03:17:00,867 [INFO] from play.api.db.HikariCPConnectionPool in application-akka.actor.internal-dispatcher-2 - Shutting down connection pool.
2021-03-06 03:17:01,064 [INFO] from play.api.db.DefaultDBApi in play-dev-mode-akka.actor.default-dispatcher-182 - Database [default] initialized
2021-03-06 03:17:01,065 [INFO] from play.api.db.HikariCPConnectionPool in play-dev-mode-akka.actor.default-dispatcher-182 - Creating Pool for datasource 'default'
2021-03-06 03:17:01,234 [INFO] from play.api.http.EnabledFilters in play-dev-mode-akka.actor.default-dispatcher-182 - Enabled Filters (see <https://www.playframework.com/documentation/latest/Filters>):

    play.filters.csrf.CSRFFilter
    play.filters.headers.SecurityHeadersFilter
    play.filters.hosts.AllowedHostsFilter

2021-03-06 03:17:01,237 [INFO] from play.api.Play in play-dev-mode-akka.actor.default-dispatcher-182 - Application started (Dev) (no global state)
2021-03-06 03:17:01,288 [ERROR] from play.api.http.DefaultHttpErrorHandler in application-akka.actor.default-dispatcher-6 - 

! @7j1ojo41f - Internal server error, for (PUT) [/recetas/] ->
 
play.api.http.HttpErrorHandlerExceptions$$anon$1: Execution exception[[PersistenceException: ManyToMany bean models.Receta@2b707b4e does not have an Id value.]]
	at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:359)
	at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:261)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:430)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:422)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:453)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:56)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:93)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:94)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:93)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:48)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:48)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
Caused by: javax.persistence.PersistenceException: ManyToMany bean models.Receta@2b707b4e does not have an Id value.
	at io.ebeaninternal.server.persist.SaveManyBeans.saveAssocManyIntersection(SaveManyBeans.java:307)
	at io.ebeaninternal.server.persist.SaveManyBeans.save(SaveManyBeans.java:69)
	at io.ebeaninternal.server.persist.DefaultPersister.saveMany(DefaultPersister.java:946)
	at io.ebeaninternal.server.persist.DefaultPersister.saveAssocMany(DefaultPersister.java:937)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:511)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:455)
	at io.ebeaninternal.server.persist.DefaultPersister.save(DefaultPersister.java:438)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1745)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1737)
	at io.ebean.Model.save(Model.java:188)
	at controllers.RecetasController.updateReceta(RecetasController.java:367)
	at router.Routes$$anonfun$routes$1.$anonfun$applyOrElse$39(Routes.scala:527)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:150)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:149)
	at play.core.routing.HandlerInvokerFactory$JavaActionInvokerFactory$$anon$3$$anon$4$$anon$5.invocation(HandlerInvoker.scala:115)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:119)
	at play.http.DefaultActionCreator$1.call(DefaultActionCreator.java:33)
	at play.core.j.JavaAction.$anonfun$apply$8(JavaAction.scala:175)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:661)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:430)
	at play.core.j.HttpExecutionContext.$anonfun$execute$1(HttpExecutionContext.scala:64)
	at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:70)
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:59)
	at scala.concurrent.impl.Promise$Transformation.submitWithValue(Promise.scala:392)
	at scala.concurrent.impl.Promise$DefaultPromise.submitWithValue(Promise.scala:302)
	at scala.concurrent.impl.Promise$DefaultPromise.dispatchOrAddCallbacks(Promise.scala:276)
	at scala.concurrent.impl.Promise$DefaultPromise.map(Promise.scala:146)
	at scala.concurrent.Future$.apply(Future.scala:661)
	at play.core.j.JavaAction.apply(JavaAction.scala:176)
	at play.api.mvc.Action.$anonfun$apply$4(Action.scala:82)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:433)
	... 11 common frames omitted
2021-03-06 03:18:44,240 [ERROR] from play.api.http.DefaultHttpErrorHandler in application-akka.actor.default-dispatcher-7 - 

! @7j1ok4096 - Internal server error, for (PUT) [/recetas/] ->
 
play.api.http.HttpErrorHandlerExceptions$$anon$1: Execution exception[[PersistenceException: ManyToMany bean models.Receta@10e8ab8d does not have an Id value.]]
	at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:359)
	at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:261)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:430)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:422)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:453)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:56)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:93)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:94)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:93)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:48)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:48)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
Caused by: javax.persistence.PersistenceException: ManyToMany bean models.Receta@10e8ab8d does not have an Id value.
	at io.ebeaninternal.server.persist.SaveManyBeans.saveAssocManyIntersection(SaveManyBeans.java:307)
	at io.ebeaninternal.server.persist.SaveManyBeans.save(SaveManyBeans.java:69)
	at io.ebeaninternal.server.persist.DefaultPersister.saveMany(DefaultPersister.java:946)
	at io.ebeaninternal.server.persist.DefaultPersister.saveAssocMany(DefaultPersister.java:937)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:511)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:455)
	at io.ebeaninternal.server.persist.DefaultPersister.save(DefaultPersister.java:438)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1745)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1737)
	at io.ebean.Model.save(Model.java:188)
	at controllers.RecetasController.updateReceta(RecetasController.java:367)
	at router.Routes$$anonfun$routes$1.$anonfun$applyOrElse$39(Routes.scala:527)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:150)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:149)
	at play.core.routing.HandlerInvokerFactory$JavaActionInvokerFactory$$anon$3$$anon$4$$anon$5.invocation(HandlerInvoker.scala:115)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:119)
	at play.http.DefaultActionCreator$1.call(DefaultActionCreator.java:33)
	at play.core.j.JavaAction.$anonfun$apply$8(JavaAction.scala:175)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:661)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:430)
	at play.core.j.HttpExecutionContext.$anonfun$execute$1(HttpExecutionContext.scala:64)
	at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:70)
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:59)
	at scala.concurrent.impl.Promise$Transformation.submitWithValue(Promise.scala:392)
	at scala.concurrent.impl.Promise$DefaultPromise.submitWithValue(Promise.scala:302)
	at scala.concurrent.impl.Promise$DefaultPromise.dispatchOrAddCallbacks(Promise.scala:276)
	at scala.concurrent.impl.Promise$DefaultPromise.map(Promise.scala:146)
	at scala.concurrent.Future$.apply(Future.scala:661)
	at play.core.j.JavaAction.apply(JavaAction.scala:176)
	at play.api.mvc.Action.$anonfun$apply$4(Action.scala:82)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:433)
	... 11 common frames omitted
2021-03-06 03:21:21,730 [ERROR] from play.api.http.DefaultHttpErrorHandler in application-akka.actor.default-dispatcher-8 - 

! @7j1okcp8f - Internal server error, for (PUT) [/recetas/] ->
 
play.api.http.HttpErrorHandlerExceptions$$anon$1: Execution exception[[PersistenceException: ManyToMany bean models.Receta@26662022 does not have an Id value.]]
	at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:359)
	at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:261)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:430)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:422)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:453)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:56)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:93)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:94)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:93)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:48)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:48)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
Caused by: javax.persistence.PersistenceException: ManyToMany bean models.Receta@26662022 does not have an Id value.
	at io.ebeaninternal.server.persist.SaveManyBeans.saveAssocManyIntersection(SaveManyBeans.java:307)
	at io.ebeaninternal.server.persist.SaveManyBeans.save(SaveManyBeans.java:69)
	at io.ebeaninternal.server.persist.DefaultPersister.saveMany(DefaultPersister.java:946)
	at io.ebeaninternal.server.persist.DefaultPersister.saveAssocMany(DefaultPersister.java:937)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:511)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:455)
	at io.ebeaninternal.server.persist.DefaultPersister.save(DefaultPersister.java:438)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1745)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1737)
	at io.ebean.Model.save(Model.java:188)
	at controllers.RecetasController.updateReceta(RecetasController.java:367)
	at router.Routes$$anonfun$routes$1.$anonfun$applyOrElse$39(Routes.scala:527)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:150)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:149)
	at play.core.routing.HandlerInvokerFactory$JavaActionInvokerFactory$$anon$3$$anon$4$$anon$5.invocation(HandlerInvoker.scala:115)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:119)
	at play.http.DefaultActionCreator$1.call(DefaultActionCreator.java:33)
	at play.core.j.JavaAction.$anonfun$apply$8(JavaAction.scala:175)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:661)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:430)
	at play.core.j.HttpExecutionContext.$anonfun$execute$1(HttpExecutionContext.scala:64)
	at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:70)
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:59)
	at scala.concurrent.impl.Promise$Transformation.submitWithValue(Promise.scala:392)
	at scala.concurrent.impl.Promise$DefaultPromise.submitWithValue(Promise.scala:302)
	at scala.concurrent.impl.Promise$DefaultPromise.dispatchOrAddCallbacks(Promise.scala:276)
	at scala.concurrent.impl.Promise$DefaultPromise.map(Promise.scala:146)
	at scala.concurrent.Future$.apply(Future.scala:661)
	at play.core.j.JavaAction.apply(JavaAction.scala:176)
	at play.api.mvc.Action.$anonfun$apply$4(Action.scala:82)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:433)
	... 11 common frames omitted
2021-03-06 03:22:25,696 [ERROR] from play.api.http.DefaultHttpErrorHandler in application-akka.actor.default-dispatcher-10 - 

! @7j1okgfok - Internal server error, for (PUT) [/recetas/] ->
 
play.api.http.HttpErrorHandlerExceptions$$anon$1: Execution exception[[PersistenceException: ManyToMany bean models.Receta@2eab2cf6 does not have an Id value.]]
	at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:359)
	at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:261)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:430)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:422)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:453)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:56)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:93)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:94)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:93)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:48)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:48)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
Caused by: javax.persistence.PersistenceException: ManyToMany bean models.Receta@2eab2cf6 does not have an Id value.
	at io.ebeaninternal.server.persist.SaveManyBeans.saveAssocManyIntersection(SaveManyBeans.java:307)
	at io.ebeaninternal.server.persist.SaveManyBeans.save(SaveManyBeans.java:69)
	at io.ebeaninternal.server.persist.DefaultPersister.saveMany(DefaultPersister.java:946)
	at io.ebeaninternal.server.persist.DefaultPersister.saveAssocMany(DefaultPersister.java:937)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:511)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:455)
	at io.ebeaninternal.server.persist.DefaultPersister.save(DefaultPersister.java:438)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1745)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1737)
	at io.ebean.Model.save(Model.java:188)
	at controllers.RecetasController.updateReceta(RecetasController.java:367)
	at router.Routes$$anonfun$routes$1.$anonfun$applyOrElse$39(Routes.scala:527)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:150)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:149)
	at play.core.routing.HandlerInvokerFactory$JavaActionInvokerFactory$$anon$3$$anon$4$$anon$5.invocation(HandlerInvoker.scala:115)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:119)
	at play.http.DefaultActionCreator$1.call(DefaultActionCreator.java:33)
	at play.core.j.JavaAction.$anonfun$apply$8(JavaAction.scala:175)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:661)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:430)
	at play.core.j.HttpExecutionContext.$anonfun$execute$1(HttpExecutionContext.scala:64)
	at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:70)
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:59)
	at scala.concurrent.impl.Promise$Transformation.submitWithValue(Promise.scala:392)
	at scala.concurrent.impl.Promise$DefaultPromise.submitWithValue(Promise.scala:302)
	at scala.concurrent.impl.Promise$DefaultPromise.dispatchOrAddCallbacks(Promise.scala:276)
	at scala.concurrent.impl.Promise$DefaultPromise.map(Promise.scala:146)
	at scala.concurrent.Future$.apply(Future.scala:661)
	at play.core.j.JavaAction.apply(JavaAction.scala:176)
	at play.api.mvc.Action.$anonfun$apply$4(Action.scala:82)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:433)
	... 11 common frames omitted
2021-03-06 03:22:48,206 [ERROR] from play.api.http.DefaultHttpErrorHandler in application-akka.actor.default-dispatcher-10 - 

! @7j1okhn6f - Internal server error, for (PUT) [/recetas/] ->
 
play.api.http.HttpErrorHandlerExceptions$$anon$1: Execution exception[[PersistenceException: ManyToMany bean models.Receta@35b8ab4a does not have an Id value.]]
	at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:359)
	at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:261)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:430)
	at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:422)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:453)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:56)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:93)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:94)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:93)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:48)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:48)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
Caused by: javax.persistence.PersistenceException: ManyToMany bean models.Receta@35b8ab4a does not have an Id value.
	at io.ebeaninternal.server.persist.SaveManyBeans.saveAssocManyIntersection(SaveManyBeans.java:307)
	at io.ebeaninternal.server.persist.SaveManyBeans.save(SaveManyBeans.java:69)
	at io.ebeaninternal.server.persist.DefaultPersister.saveMany(DefaultPersister.java:946)
	at io.ebeaninternal.server.persist.DefaultPersister.saveAssocMany(DefaultPersister.java:937)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:511)
	at io.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:455)
	at io.ebeaninternal.server.persist.DefaultPersister.save(DefaultPersister.java:438)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1745)
	at io.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1737)
	at io.ebean.Model.save(Model.java:188)
	at controllers.RecetasController.updateReceta(RecetasController.java:367)
	at router.Routes$$anonfun$routes$1.$anonfun$applyOrElse$39(Routes.scala:527)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:150)
	at play.core.routing.HandlerInvokerFactory$$anon$8.resultCall(HandlerInvoker.scala:149)
	at play.core.routing.HandlerInvokerFactory$JavaActionInvokerFactory$$anon$3$$anon$4$$anon$5.invocation(HandlerInvoker.scala:115)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:119)
	at play.http.DefaultActionCreator$1.call(DefaultActionCreator.java:33)
	at play.core.j.JavaAction.$anonfun$apply$8(JavaAction.scala:175)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:661)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:430)
	at play.core.j.HttpExecutionContext.$anonfun$execute$1(HttpExecutionContext.scala:64)
	at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:70)
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:59)
	at scala.concurrent.impl.Promise$Transformation.submitWithValue(Promise.scala:392)
	at scala.concurrent.impl.Promise$DefaultPromise.submitWithValue(Promise.scala:302)
	at scala.concurrent.impl.Promise$DefaultPromise.dispatchOrAddCallbacks(Promise.scala:276)
	at scala.concurrent.impl.Promise$DefaultPromise.map(Promise.scala:146)
	at scala.concurrent.Future$.apply(Future.scala:661)
	at play.core.j.JavaAction.apply(JavaAction.scala:176)
	at play.api.mvc.Action.$anonfun$apply$4(Action.scala:82)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:433)
	... 11 common frames omitted
2021-03-06 03:27:05,553 [INFO] from play.api.db.HikariCPConnectionPool in application-akka.actor.internal-dispatcher-11 - Shutting down connection pool.
2021-03-06 03:27:05,754 [INFO] from play.api.db.DefaultDBApi in play-dev-mode-akka.actor.default-dispatcher-214 - Database [default] initialized
2021-03-06 03:27:05,754 [INFO] from play.api.db.HikariCPConnectionPool in play-dev-mode-akka.actor.default-dispatcher-214 - Creating Pool for datasource 'default'
2021-03-06 03:27:05,920 [INFO] from play.api.http.EnabledFilters in play-dev-mode-akka.actor.default-dispatcher-214 - Enabled Filters (see <https://www.playframework.com/documentation/latest/Filters>):

    play.filters.csrf.CSRFFilter
    play.filters.headers.SecurityHeadersFilter
    play.filters.hosts.AllowedHostsFilter

2021-03-06 03:27:05,922 [INFO] from play.api.Play in play-dev-mode-akka.actor.default-dispatcher-214 - Application started (Dev) (no global state)
2021-03-06 03:35:11,225 [INFO] from play.api.db.HikariCPConnectionPool in application-akka.actor.internal-dispatcher-7 - Shutting down connection pool.
2021-03-06 03:35:11,403 [INFO] from play.api.db.DefaultDBApi in play-dev-mode-akka.actor.default-dispatcher-225 - Database [default] initialized
2021-03-06 03:35:11,403 [INFO] from play.api.db.HikariCPConnectionPool in play-dev-mode-akka.actor.default-dispatcher-225 - Creating Pool for datasource 'default'
2021-03-06 03:35:11,564 [INFO] from play.api.http.EnabledFilters in play-dev-mode-akka.actor.default-dispatcher-225 - Enabled Filters (see <https://www.playframework.com/documentation/latest/Filters>):

    play.filters.csrf.CSRFFilter
    play.filters.headers.SecurityHeadersFilter
    play.filters.hosts.AllowedHostsFilter

2021-03-06 03:35:11,567 [INFO] from play.api.Play in play-dev-mode-akka.actor.default-dispatcher-225 - Application started (Dev) (no global state)
